content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,505 @@
+setting once an appraisal is confirmed). The toggle changes to :guilabel:`Visible to Manager`.
+
+.. _appraisals/manager-feedback:
+
+Manager's feedback
+******************
+
+After the employee has completed the :guilabel:`Employee's Feedback` section, it is time for the
+manager to fill out the :guilabel:`Manager's Feedback` section. The manager enters the responses in
+the fields in the :ref:`same manner as the employee <appraisals/employee-feedback>`. When the
+feedback section is completed, click the :guilabel:`Not Visible to Employee` toggle (the default
+setting once an appraisal is confirmed). The toggle changes to :guilabel:`Visible to Employee`.
+
+.. image:: appraisals/feedback.png
+   :align: center
+   :alt: The feedback section for both employees and managers. The toggle buttons are highlighted.
+
+Skills tab
+~~~~~~~~~~
+
+Part of an appraisal is evaluating an employee's skills and tracking their progress over time. The
+:guilabel:`Skills` tab of the appraisal auto-populates with the skills from the :doc:`employee form
+<../hr/employees/new_employee>` once an appraisal is confirmed.
+
+Each skill is grouped with like skills, and the :guilabel:`Skill Level`, :guilabel:`Progress`, and
+:guilabel:`Justification` are displayed for each skill. Update any skills, or add any new skills to
+the skills tab. If a skill level has increased, a reason for the improved rating can be entered into
+the :guilabel:`Justification` field, such as `took a fluency language test` or `received Javascript
+certification`. Refer to the :ref:`Create a new employee <employees/skills>` document for detailed
+instructions on adding or updating a skill.
+
+After an appraisal is completed and the skills have been updated, the next time an appraisal is
+confirmed, the updated skills populate the skills tab.
+
+.. image:: appraisals/skills.png
+   :align: center
+   :alt: The skills tab of an appraisal form, all filled out.
+
+Private note tab
+~~~~~~~~~~~~~~~~
+
+If managers want to leave notes that are only visible to other managers, they can be entered in the
+:guilabel:`Private Note` tab. The employee being evaluated does not have access to this tab, and the
+tab does not appear on their appraisal.
+
+Schedule a meeting
+------------------
+
+Once both portions of an appraisal are completed (the :ref:`employee
+<appraisals/employee-feedback>` and :ref:`manager <appraisals/manager-feedback>` feedback sections),
+it is time for the employee and manager to meet and to discuss the appraisal. A meeting can be
+scheduled in one of two ways: either from the *Appraisals* application dashboard, or from an
+individual appraisal card.
+
+To schedule an appraisal from the dashboard of the *Appraisals* application, first navigate to
+:menuselection:`Appraisals application --> Appraisals`.
+
+Click on the activity icon beneath the appraisal date on the desired appraisal card, and an activity
```

Comment:
Originally, I didn't think so since it was an icon and not the proper name, but now that you mention it, it is something you click, so YES. I changed my mind- agreed, and I am changing it in both places.